### ISSUE TITLE
Where are the Scheme communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/scheme#77]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Scheme supported by one or more large organizations?
* Does Scheme have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Scheme? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Scheme? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Scheme? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Scheme?
* Are there popular newsletters for Scheme?
* Is Scheme taught at programming bootcamps? (If so, what are some examples?)
* Is Scheme taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Scheme?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by panicz** [comment]
**Is Scheme supported by one or more large organizations?**

It depends on what you mean by "support".
Some time ago, Cisco purchased Cadence, the developer of the top quality Scheme implementation called Chez. Since then, they released the compiler to the [open source](https://github.com/cisco/chezscheme).
Microsoft recently also released a low footprint implementation of Scheme for the .NET platform called [Schemy](https://github.com/Microsoft/schemy).

**Does Scheme have an official community manager?**

It's a complex issue. The Scheme community isn't just a community of language users, but also a community of language implementors. The development of Scheme proceeds in a highly democratic fashion. The development of the subsequent Scheme standards is entirely community-driven, and the extension proposals (Scheme Request for Implementation or [SRFI](https://srfi.schemers.org/) need to pass a formal review process.
The current role of SRFI editor is played by [Arthur Gleckler](https://speechcode.com/), and the person who has been driving the most recent Scheme standard ([R7RS](https://www.r7rs.org/)) is [John Cowan](http://vrici.lojban.org/~cowan/).
Note however, that the people who play these roles have been changing historically.

**Do you know of specific communities (online or offline) that are enthusiastic about Scheme? (Chat communities, forums, meetups, student clubs, etc)**

[schemers.org](http://schemers.org/) is probably the most centralized resource for the diverse Scheme community. There are also vivid communities around various particular implementations such as [Racket](https://racket-lang.org/), [Chicken](https://www.call-cc.org/), [Guile](https://www.gnu.org/software/guile/), [Gambit](http://gambitscheme.org/wiki/index.php/Main_Page) and [many others](http://community.schemewiki.org/?scheme-faq-standards#implementations).
There is also [comp.lang.scheme](https://groups.google.com/forum/#!forum/comp.lang.scheme) news group on usenet and a [reddit room](https://www.reddit.com/r/scheme/).

Some meetups are pointed to by [schemers.org](http://schemers.org/), but I don't think that the list is updated very often.

**Are there popular conferences for Scheme? (If so, what are some examples?)**

The most important one is [Scheme Workshop](http://www.schemeworkshop.org/), which is held annually with [ICFP](https://www.icfpconference.org/). Note however, that the word "intimate" describes it better than "popular", especially if you compare it with conferences devoted to languages like Python or C++.

**Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Scheme? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)**

There is [Bootstrap](http://www.bootstrapworld.org/) which is targeted mainly at kids.

**Are there specific groups or programs dedicated to mentoring people in Scheme?**

Not that I know of. But most Schemers I know are really kind and helpful. I've seen some people devoting their time to answer Scheme-related questions on Stack Overflow.

**Are there popular newsletters for Scheme?**

I don't think so.

**Is Scheme taught at programming bootcamps? (If so, what are some examples?)**

All the bootcamps I'm familiar with focus on teaching technologies that are useful for jobs for which there is a high demand on the market - which in my experience mainly means web-related things like JavaScript, HTML, CSS or SQL, and frameworks like ReactJS. While JavaScript is a cluttered derivative of Scheme, it seems that teaching Scheme in such short-sighted contexts would be an expensive luxury.

**Is Scheme taught at universities? (If so, what are some examples?)**

Yes. Scheme has been used for many years by MIT in their introductory [Computer Science course](https://mitpress.mit.edu/sites/default/files/sicp/index.html).
Currently it is also a significant part of the curricula of [Northeastern](https://www.ccis.northeastern.edu/), [Brown](http://cs.brown.edu/), [Indiana](https://www.cs.indiana.edu/) and [WPI](https://www.wpi.edu/academics/departments/computer-science), among others.

**Comment by kytrinyx** [comment]
@panicz Thank you so much, this is incredibly helpful!

I'm going to close this, as I think we've got what we need. 💙

**Comment by panicz** [comment]
There are also some Scheme fans on Quora and Twitter.
It's easy to find them on Quora in particular:
https://www.quora.com/topic/Scheme-programming-language-1/writers

As to Twitter, I can recommend the following accounts:
https://twitter.com/racketlang
https://twitter.com/guilelang
https://twitter.com/alamajesse
https://twitter.com/ShriramKMurthi
https://twitter.com/lambda_calculus
https://twitter.com/mattmight
https://twitter.com/webyrd
https://twitter.com/schemeworkshop
https://twitter.com/nadamin
https://twitter.com/sperbsen
https://twitter.com/andywingo
https://twitter.com/dustyweb
https://twitter.com/dfried00
https://twitter.com/mwand
https://twitter.com/lexi_lambda
https://twitter.com/samth

**Comment by kytrinyx** [comment]
That's a great find, thank you.
